src/main/java/gift/config/JwtProvider.java

```diff
@@ -0,0 +1,25 @@
+package gift.config;
+
+import gift.entity.Member;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import org.springframework.stereotype.Component;
+
+
+@Component
+public class JwtProvider {
+
+
+    private final String secretKey = "Yn2kjibddFAWtnPJ2AFlL8WXmohJMCvigQggaEypa5E=";
+
+
+    public String generateToken(Member member) {
+        String accessToken = Jwts.builder()
+                .setSubject(member.getId().toString())
+                .claim("email", member.getEmail())
+                .signWith(Keys.hmacShaKeyFor(secretKey.getBytes()))
+                .compact();
+
+        return accessToken;
```

Comment:
```suggestion
        return Jwts.builder()
                .setSubject(member.getId().toString())
                .claim("email", member.getEmail())
                .signWith(Keys.hmacShaKeyFor(secretKey.getBytes()))
                .compact();
```

변수를 활용하지 않으니 inline 으로 사용하는걸 어떨까요?